src/Rings/mpoly-localizations.jl

```diff
@@ -255,8 +255,6 @@ function prime_ideal(S::MPolyComplementOfKPointIdeal)
   return S.m
 end
 
-dim(kk::Field) = 0
```

Comment:
you also need to adjust the code just a few lines above this to call `krull_dim` instead of `dim`...